### ISSUE TITLE
Issue Reporter: show a good issue title example for placeholder

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#211126]
Testing #210874

Could we do better than `Please enter a title.`? Perhaps an example that feels somewhat realistic to convey what a good issue title looks like? `E.g. missing alt text on extension readme image` for an extension issue. We could use different ones for each source.

![Image](https://github.com/microsoft/vscode/assets/25163139/bc2fc98f-b42d-4be9-8e13-296759d5b610)

**Comment by justschen** [comment]
@daviddossett i like this too. kind of related to https://github.com/microsoft/vscode/issues/210739 where we might want to specify some things in the drop down as well

**Comment by justschen** [comment]
brainstorming here for sources:

maybe something like this? these are just random issues i pulled or made up

VS Code: `E.g. Workbench missing problems panel`
Extension: `E.g. Missing alt text on extension readme image`
Marketplace: `E.g Cannot disable installed extension`
